Automattic/woocommerce-payments
# E2E tests for checking out with Giropay

Fixes #6761 

#### Changes proposed in this Pull Request

* Add an E2E to test the [Giropay checkout](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows#giropay-checkout) experience.
* Fix checkbox selectors for UPE PMs (broke after the introduction of Affirm/Afterpay).

#### Testing instructions

* Verify that all E2E tests are passing in GitHub actions.
* Feeling adventurous? Follow [this guide](https://github.com/Automattic/woocommerce-payments/tree/develop/tests/e2e) to locally set up the environment to run E2E tests.
  * Run the spec for deferred UPE, which contains the tests for Giropay. Ensure that the new tests are passing.
  * `npm run test:e2e-dev tests/e2e/specs/upe-split/shopper/shopper-deferred-intent-creation-upe-enabled.spec.js`

-------------------


**Post merge**
